server/src/main/java/org/apache/gravitino/server/web/rest/ModelOperations.java

```diff
@@ -516,11 +666,20 @@ public Response alterModelVersionByAlias(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "alter-model." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "alter-model", absolute = true)
+  @AuthorizationExpression(
+      expression =
+          "ANY(OWNER,METALAKE,CATALOG,SCHEMA) || "
```

Comment:
SchemaOwner should have privilege `use  catalog`.

Reply:
fixed